client/src/app/pages/applications/application-assessment/components/assessment-summary/assessment-summary-page.tsx

```diff
@@ -0,0 +1,201 @@
+import React, { useEffect, useState, useMemo } from "react";
+import yaml from "js-yaml";
+import {
+  Text,
+  TextContent,
+  PageSection,
+  PageSectionVariants,
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  Tabs,
+  Toolbar,
+  ToolbarItem,
+  SearchInput,
+  ToolbarContent,
+  Tab,
+} from "@patternfly/react-core";
+import AngleLeftIcon from "@patternfly/react-icons/dist/esm/icons/angle-left-icon";
+import { Assessment } from "@app/api/models";
+import { Link, useParams } from "react-router-dom";
+import { Paths, formatPath } from "@app/Paths";
+import { ConditionalRender } from "@app/components/ConditionalRender";
+import { AppPlaceholder } from "@app/components/AppPlaceholder";
+import { useTranslation } from "react-i18next";
+import QuestionnaireSectionTabTitle from "./components/questionnaire-section-tab-title";
+import "./assessment-summary-page.css";
+import { useFetchQuestionnaires } from "@app/queries/questionnaires";
+import { useFetchAssessmentById } from "@app/queries/assessments";
+import QuestionsTable from "@app/components/questions-table/questions-table";
+
+interface AssessmentSummaryRouteParams {
+  assessmentId: string;
+  applicationId: string;
+}
+
+const AssessmentSummaryPage: React.FC = () => {
+  const { assessmentId } = useParams<AssessmentSummaryRouteParams>();
+
+  const { t } = useTranslation();
+
+  const [activeSectionIndex, setActiveSectionIndex] = React.useState<
+    "all" | number
+  >("all");
+
+  const handleTabClick = (
+    _event: React.MouseEvent<any> | React.KeyboardEvent | MouseEvent,
+    tabKey: string | number
+  ) => {
+    setActiveSectionIndex(tabKey as "all" | number);
+  };
```

Comment:
Can we create a component shared by `AssessmentSummaryPage` and `QuestionnairePage` for the layout of tabs and search logic instead of copying that code here? I think maybe there's a higher level of stuff we can reuse here than just the QuestionsTable. We could create something like a `QuestionnaireSummary` that can handle either the questionnaire itself or an assessment with conditional logic for the parts that differ.

If that's too much hassle for this PR we could do it in a followup.